kerckhoff/packages/views.py

```diff
@@ -1,20 +1,27 @@
+from multiprocessing import log_to_stderr
+from typing import List
+import os
+import json 
+from importlib_metadata import packages_distributions
```

Comment:
There's a bunch of unused imports here. I think again it's from the IDE

Reply:
Removed extra imports.